### PR TITLE
Add feature tests

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Comment by wchrapka** [comment]
IMO testing it for cluster and namespace falls onto DRY (testing it twice). 

I would test if app behaves normally, if a flag like "PROMETHEUS" is not defined at all, against "reading isEnabled of undefined"

Are we treating other flags separately?

File: tests/tests/cluster/test-navigation-features.spec.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    // visual resources
+    cy.navigateTo('Configuration', 'Cluster Role Bindings');
+
+    cy.getIframeBody()
+      .contains('cluster-admin')
+      .click();
+
+    // wait for view to stabilize
+    cy.wait(5000);
```
Comment (wchrapka) [inline tests/tests/cluster/test-navigation-features.spec.js:49]: maybe 

```suggestion
      cy.getIframeBody()
      .contains('system:masters') // means the view is rendered
      .should('be.visible');
```

Comment (Wawrzyn321) [reply tests/tests/cluster/test-navigation-features.spec.js:49]: I get it, but for me it navigates to cluster roles... I'll try to find a better selector.

**Review by wchrapka — APPROVED** [review]

**Pull request merged**